specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2019-03-18/HybridCompute.json

```diff
@@ -0,0 +1,789 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HybridComputeManagementClient",
+    "description": "The Hybrid Compute Management Client.",
+    "version": "2019-01-01-preview"
```

Comment:
API version doesn't align with the one specified in folder name. And please have only one API version swagger in a PR.

Reply:
Updated.